baybe/targets/binary.py

```diff
@@ -0,0 +1,91 @@
+"""Binary targets."""
+
+from typing import TypeAlias
+
+import numpy as np
+import pandas as pd
+from attrs import define, field
+from attrs.validators import instance_of
+
+from baybe.exceptions import InvalidTargetValueError
+from baybe.serialization import SerialMixin
+from baybe.targets.base import Target
+
+ChoiceValue: TypeAlias = bool | int | float | str
+"""Types of values that a :class:`BinaryTarget` can take."""
+
+_POSITIVE_VALUE_COMP = True
+"""Computational representation of the positive choice value."""
+
+_NEGATIVE_VALUE_COMP = False
+"""Computational representation of the negative choice value."""
+
+
+@define(frozen=True)
+class BinaryTarget(Target, SerialMixin):
```

Comment:
I think this somehow means that in various places where we have explicitly type hinted `NumericalTarget` this should actually now be `Target` to allow both variants

Reply:
the update does not have to be part of this PR but should be remembered, resolve

Reply:
yes, already aware of it. one of the items in my head  💇🏼‍♂️